develop/blockchains/deployment/generate-chain-specs.md

```diff
@@ -131,6 +131,6 @@ The raw chain specification can be used to initialize the genesis storage for a
 
 After generating a chain specification, you can use it to initialize the genesis storage for a node. Refer to the following guides to learn how to proceed with the deployment of your blockchain:
 
-- [**Obtain Coretime**](TODO:update-path) - learn how to obtain the necessary coretime configuration to synchronize your blockchain’s timestamping and enhance its performance
-- [**Deployment**](TODO:update-path) - explore the steps required to deploy your chain specification, ensuring a smooth launch of your network and proper node operation
-- [**Maintenance**](TODO:update-path) - discover best practices for maintaining your blockchain post-deployment, including how to manage upgrades and monitor network health
\ No newline at end of file
+- [**Obtain Coretime**](/tutorials/blockchains/custom-blockchains/build-a-parachain/acquire-a-testnet-slot/#obtain-coretime){target=\_blank} - learn how to obtain the necessary coretime configuration to synchronize your blockchain’s timestamping and enhance its performance
```

Comment:
I fixed a few links here, just double check that the obtain coretime is the one you want! @nhussein11

Reply:
This works for now. Added a comment with a todo to change it once we have an obtain coretime guide